{"name": "test_getbyname_failure", "status": "passed", "start": 1729274488680, "stop": 1729274488681, "uuid": "417f195f-73fc-4aca-a8c5-d91ad4442c98", "historyId": "da79b9892db5ab531f3b11a7f1273cec", "testCaseId": "da79b9892db5ab531f3b11a7f1273cec", "fullName": "Exercise.exercise_test.TestExerciseServiceWithMock#test_getbyname_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "7844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}